launcher/ui/dialogs/ProfileSetupDialog.ui

```diff
@@ -30,6 +30,9 @@ Choose your name carefully:</string>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
+     <property name="textInteractionFlags">
+      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
```

Comment:
TextSelectableByKeyboard is a little inconsistent with other dialog boxes, we usually just have TextSelectableByMouse

Reply:
Yeah but still: is this a big deal?
I do not have a strong opinion on this, so if you want the change, please make my life easier and suggest the change.